### ISSUE TITLE
"Drag to Applications/ to install"

### ISSUE THREAD

**Issue by hoosierEE** [pilotmoon/Scroll-Reverser#16]
Might be useful to add the dialog box which prompts the user to "drag this icon to the Applications folder" either as part of the install process, or at least make it more prominent in the install instructions.

Nice app by the way, does just what I wanted. :+1:

**Comment by invariant** [comment]
Thanks for suggestion. I'm not convinced of the need for it, as SR will run happily whatever folder it is in, so there is no need to move it to and Applications folder. Although that is of course the usual convention.

**Comment by hoosierEE** [comment]
For me the process was something like:

1. download (saved to ~/Downloads)
2. unzip and double-click icon to install
3. delete contents of ~/Downloads
4. reboot computer... Scroll Reverser is gone!

That's when I realized that the install scripts did not magically place the app somewhere more permanent.  This is, of course, a Good Thing.  But it's not explicitly stated anywhere in the docs.

In lieu of the OSX convention of "drag to Applications folder", perhaps a footnote about how to install?

**Comment by hoosierEE** [comment]
Meh, I guess it's not exactly a pressing issue. Closing!

**Comment by invariant** [comment]
Hehe, it's OK to leave it open as a reminder if you like. TBH I haven't even thought about it, so much other stuff to do ... (such is life)

**Comment by hoosierEE** [comment]
Nah, I'd forgotten about it, but noticed when I spuriously checked the "issues" tab in my own GitHub profile.  

Regardless, someone who wants to reverse their scroll direction differently for different devices is likely *also* the kind of person who can figure out how to install it. :+1:

**Comment by SystemDisc** [comment]
It's still a good idea to have an intuitive UI. Most apps you download for Mac have something like this:
<img src="https://i.imgur.com/nHRelRK.png">
<img src="https://i.imgur.com/oehkrI5.png">

**Comment by hoosierEE** [comment]
@SystemDisc thanks for that link, that's easier than I thought it would be.

**Comment by latenitefilms** [comment]
This might also be of interest:

https://github.com/potionfactory/LetsMove
